ingest-app/src/cmr/ingest/site/routes.clj

```diff
@@ -0,0 +1,47 @@
+(ns cmr.ingest.site.routes
+  "Defines the HTTP URL routes for the ingest web site."
+  (:require
+   [cmr.common-app.static :as static]
+   [cmr.common-app.site.data :as common-data]
+   [cmr.common-app.site.pages :as common-pages]
+   [cmr.ingest.site.pages :as pages]
+   [cmr.transmit.config :as config]
+   [compojure.core :refer [GET context routes]]
+   [ring.util.response :refer [redirect]]))
+
+(defn build-routes [system]
+  (routes
+    (context (get-in system [:ingest-public-conf :relative-root-url]) []
+      (GET "/"
+           {ctx :request-context}
+           (pages/home ctx))
```

Comment:
It's `lein generate-static` now, but yes -- that will generate all static content we have set up to be generated (i.e., added to the subprojects `site.static` ns). That should be possible with all content that doesn't depend upon site-specific configuration data, db data, etc.

We will need to split out the base-static data to distinguish between levels for relative URLs and how the `base-url` is generated for static content (this was originally done via each app's `public-conf`, but has been updated for static content to use a relative base).